client/modules/reconportal/src/ReconSidePanel/ContributeDataModal.tsx

```diff
@@ -0,0 +1,287 @@
+import React, { useState } from 'react';
+import { Button, Form, Input, Modal, Typography, notification } from 'antd';
+import { Formik, Form as FormikForm } from 'formik';
```

Comment:
For consistency with the rest of the codebase, should we use Antd Form + Zod instead of Formik + Yup?